core/driver/sources/src/main/resources/reference.conf

```diff
@@ -5,12 +5,52 @@ probe {
 
   // `intellij` is for org.virtuslab.ideprobe.config.IntellijConfig - default values below:
   intellij {
+    // `plugins` array contains IntelliJ plugins which should be installed and enabled on IntelliJ run. Here you can add
+    // your IntelliJ plugin for tests (amongst other plugins). There are a few ways to provide a proper entry pointing
+    // to a plugin:
+    // 1. By using a direct URI pointing to the plugin (could be from a website or from a local file). For example:
+    #     { uri = "https://packages.jetbrains.team/maven/p/ij/intellij-dependencies/com/intellij/remoterobot/robot-server-plugin/0.11.15/robot-server-plugin/0.11.15.zip" }
+    // 2. By using default IntelliJ plugins repository (see `probe.resolvers.plugins.repository.uri`) with additional
+    //    id (pluginId), version and (optional) channel parameters which is translated into a URL with a query string.
+    //    For example:
+    #       { id = "org.intellij.scala", version = "2020.3.369", channel = "nightly" }
+    //      will be resolved into following URL with query string (from where ide-probe should download the plugin):
+    //      https://plugins.jetbrains.com/plugin/download?pluginId=org.intellij.scala&version=2020.3.369&channel=nightly
+    //    and without the optional "channel" parameter:
+    #       { id = "org.intellij.scala", version = "2020.3.369" }
+    //      will be resolved into following URL with query string (from where ide-probe should download the plugin):
+    //      https://plugins.jetbrains.com/plugin/download?pluginId=org.intellij.scala&version=2020.3.369
+    // 3. By using a bundled IntelliJ plugin. Such plugin should be bundled in your project as a module/subproject and
+    //    available on the classpath when IntelliJ instance should run. For example - such plugin subproject should be
+    //    built and published to a `target` directory before performing tests (during compilation).
+    //    There are two ways to point to a bundled IntelliJ plugin:
+    //    a) By providing plugin's resource file name (probably plugin subproject's name followed by version and file extension).
+    //       For example:
+    #        { bundle = "driver-test-plugin_2.13-0.45.0-SNAPSHOT.zip" }
```

Comment:
And the use of `bundle` in config also isn't fortunate... but it's a different story 😅

Reply:
Huh unless `bundled` here and `bundled` as in, _included with IntelliJ by default_ somehow correspond to the same category of plugins? 🤔 but it isn't obvious

Reply:
But now I see that it leads to confusion. Maybe a short introduction to ide-probe's _bundle_ naming convention will be needed here?

Reply:
Yeah, pls open an issue for the ☝🏻